plugins/widget/plugin.js

```diff
@@ -4253,6 +4313,17 @@
  * @property {String} button
  */
 
+/**
+ * Customizes widget HTML copied to the clipboard
```

Comment:
Isn't it duplicating `getClipboardHtml()` method defined above (https://github.com/ckeditor/ckeditor-dev/pull/3168/files#diff-d932030cd4e22ec8b0e30a362269b7e8R1244)? 🤔

Reply:
One is defined directly on `CKEDITOR.plugins.widget` class and one on widget's definition.